src/focus-ring.js

```diff
@@ -29,22 +31,31 @@ document.addEventListener('DOMContentLoaded', function() {
 	    }
 	    return triggers;
 	},
+        _addClass = function(el, htmlClass) {
```

Comment:
because we're declaring these inside of a function I don't think you need to underscore their names